core/src/main/java/org/elasticsearch/ElasticsearchException.java

```diff
@@ -580,8 +579,7 @@ public String toString() {
                 org.elasticsearch.action.TimestampParsingException::new, 78),
         ROUTING_MISSING_EXCEPTION(org.elasticsearch.action.RoutingMissingException.class,
                 org.elasticsearch.action.RoutingMissingException::new, 79),
-        OPERATION_FAILED_ENGINE_EXCEPTION(OperationFailedEngineException.class,
-                OperationFailedEngineException::new, 80),
+        // 80 used to be for IndexFailedEngineException, removed in 6.0
```

Comment:
@bleskes we need bwc here if we care about rolling restarts from 5.x. I am not sure how to add bwc for this (i.e. might be missing something). We could simply add the exceptions back (but not use them anywhere) so they can be serialized/deserialized when a 6.0 node is acting as a coordinating node? 

Another question: The index/delete operation failures were communicated as exceptions, so do we even need a bwc for these failures for serialization/deserialization or can we just rely on generic exception serialization/deserialization like we currently do for persistent engine failures during index/delete operations?